### PR TITLE
Addition of Avalanche Chain IDs

### PR DESCRIPTION
**Pull request by kylecbetts**
Avalanche is an open-source platform for launching decentralized applications and enterprise blockchain deployments in one interoperable, highly scalable ecosystem.

Avalanche uses two chain IDs:
43113 for Avalanche Fuji Testnet
43114 for Avalanche Mainnet

### CODE REVIEW

File: _data/chains/epi155-43114.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "https://api.avax.network/ext/bc/C/rpc"
+    ],
+    "faucets": [],
+    "nativeCurrency": {
+      "name": "AVAX",
+      "symbol": "AVAX",
+      "decimals": 18
+    },
+    "infoURL": "https://cchain.explorer.avax.network/",
+    "shortName": "Mainnet",
```
Comment (dasconnor) [inline _data/chains/epi155-43114.json:15]: Should be `Avalanche`

File: _data/chains/eip155-43113.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Avalanche Testnet Fuji",
```
Comment (dasconnor) [inline _data/chains/eip155-43113.json:2]: `Avalanche Fuji Testnet`

File: _data/chains/eip155-43113.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    ],
+    "nativeCurrency": {
+      "name": "AVAX",
+      "symbol": "AVAX",
+      "decimals": 18
+    },
+    "infoURL": "https://cchain.explorer.avax-test.network",
+    "shortName": "Fuji",
+    "chainId": 43113,
+    "networkId": 43113
```
Comment (dasconnor) [inline _data/chains/eip155-43113.json:19]: networkId is 1

File: _data/chains/epi155-43114.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "chainId": 43114,
+    "networkId": 43114
```
Comment (dasconnor) [inline _data/chains/epi155-43114.json:17]: networkId is 1

File: _data/chains/eip155-43113.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Avalanche Fuji Testnet",
+    "chain": "AVAX",
+    "network": "testnet",
+    "rpc": [
+      "https://api.avax-test.network/ext/bc/C/rpc"
+    ],
+    "faucets": [
+        "https://faucet.avax-test.network/"
```
Comment (cgcardona) [inline _data/chains/eip155-43113.json:9]: Minor nit—this line is indented 4 spaces and the rest of the file uses 2 spaces for indentations

Comment (cgcardona) [reply _data/chains/eip155-43113.json:9]:
```json
    "faucets": [
        "https://faucet.avax-test.network/"
```

Should be

```json
    "faucets": [
      "https://faucet.avax-test.network/"
```

**Review by ligi — CHANGES_REQUESTED** [review]
there is a typo in the filename: epi155-43114.json
Must be: eip155-43114.json

**Review by ligi — APPROVED** [review]

**Pull request merged**